asmcli/commands/experimental/vm.sh

```diff
@@ -321,7 +321,7 @@ is_google_identity_provider_installed() {
 }
 
 is_idp_crd_installed() {
-  if ! kubectl api-resources --api-group=security.cloud.google.com | grep -q identityproviders; then
+  if ! retry 2 kubectl api-resources --api-group=security.cloud.google.com | grep -q identityproviders; then
```

Comment:
If this can't use `get crd` and requires a `| grep` then let's split it into two steps.

```
local OUTPUT; OUTPUT="$(retry 2 kubectl api-resources...)"
if ! echo "${OUTPUT}" | grep -q identityproviders; then
```

Reply:
can I use https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/main/asmcli/asmcli#L1914-L1915? like

```
is_membership_crd_installed() {
  if [[ "$(retry 2 kubectl get crd memberships.hub.gke.io -ojsonpath="{..metadata.name}" \
    | grep -w -c memberships || true)" -eq 0 ]]; then
    false
    return
  fi

  if [[ "$(retry 2 kubectl get memberships.hub.gke.io -ojsonpath="{..metadata.name}" \
    | grep -w -c membership || true)" -eq 0 ]]; then
    false
  fi
}
```

Reply:
That should also work, yea.

Reply:
Done